lib/commons/aria/implicit-role.js

```diff
@@ -25,16 +38,14 @@ function implicitRole(node) {
     );
   }
 
-  // until we have proper implicit role lookups for svgs we will
-  // avoid giving them one
-  if (node && node.namespaceURI === 'http://www.w3.org/2000/svg') {
-    return null;
-  }
```

Comment:
It looked like an unnecessary check - we don't have implicit role lookups for svg, so it will return null without this check based on the lines that follow. 

If we left it in, we'd have to add another check here for whether to use the chromium roles, and return the svg's implicit chromium role or else null. That's the same logic as is below.